src/EventSubscriber/ErrorHandlerSubscriber.php

```diff
@@ -51,7 +51,7 @@ public function onKernelException(ExceptionEvent $event): void
         }
 
         $event->setResponse(
-            $this->responder->createErrorJsonResponse($event->getException())
+            $this->responder->createErrorJsonResponse($event->getThrowable())
```

Comment:
i dont see this method in 4.3, how is this better than `getException ` ? any benefits ?

Reply:
It's marked as deprecated in Symfony 4.4

Reply:
ok, thx :)